services/analyzer/sequelize-tables-analyzer.js

```diff
@@ -231,11 +231,14 @@ async function createTableSchema({
     const isValidField = type && (!foreignKey
       || !foreignKey.foreignTableName
       || !foreignKey.columnName || columnInfo.primaryKey);
+    // NOTICE: If the column is of integer type, named "id" and primary, Sequelize will handle it
+    //         automatically without necessary declaration.
+    const isIdIntegerPrimaryColumn = columnName === 'id'
+      && ['INTEGER', 'BIGINT'].includes(type)
```

Comment:
Here is the fix

The rest of the file change is a kind of refactoring.
Sorry to introduce this in the PR but as the refactoring is also done in a PR in Sliman's hands (https://github.com/ForestAdmin/lumber/pull/376) I wanted to reduce a bad conflict resolution.
(Even if the tests would reveal a potential bad conflict resolution)